### ISSUE TITLE
Whire Balance applied twice

### ISSUE THREAD

**Issue by denm77** [darktable-org/darktable#14501]
Hi, 
Opening any NEF files ( just after import ) now the white balance is applied twice.Wich was not at least since version 4.2.1 Install.
I need everytime to  click on the BULB symbol to fix this and WB is then correctly applied.
Anyone could help with that small issue ?

Thanks

**Comment by gi-man** [comment]
Please, use the template to provide all the relevant information (eg. what version/build).

**Comment by ptilopteri** [comment]
and attach a representative sidecar file (xmp).

**Comment by MStraeten** [comment]
That’s default behaviour when using modern whitebalance, since color calibration prerequisites the camera reference setting in white balance.

**Comment by ralfbrown** [comment]
> That’s default behaviour when using modern whitebalance, since color calibration prerequisites the camera reference setting in white balance.

True, but if this is an old edit and the camera reference value has been changed in the code since, we could have this situation.  Which is why @denm77 needs to tell us which camera model, which version of darktable it occurs on, etc.  And provide a sidecar file for further analysis.

**Comment by Nilvus** [comment]
Duplicate on multiple issues. See #12134, #12801 and #11377 and probably others. As a duplicate and you will have all infos on linked issues, I close yours.
